### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by opolsen**

### CODE REVIEW

**Comment by joakimmj** [comment]
Litt uenig i valget om rekkefølge på tuple fra split funksjon. For det første er det mer naturlig å dele først data, etterfulgt av labels.. Istedenfor å dele og mikse. For det andre er det også greiere for de som velger å bruke sklearn.model_selection.train_test_split

**Comment by haakondi** [comment]
Da var jeg kanskje litt rask med å merge denne :p

**Comment by opolsen** [comment]
Jeg er forsåvidt enig i det, men det burde være konsistent i begge
oppgavene. Hva med å endre til rekkefølgen du foreslo begge stedene?

**Comment by joakimmj** [comment]
Jeg trudde jeg lagde en kommentar på det tidligere i dag, men ser ikke ut til at den er her :P

**Comment by joakimmj** [comment]
Også veldig enig i at vi bør være konskvente med rekkefølge.. Jeg foreslo i kommentaren min at vi byttet :P
